kuma/javascript/src/__mocks__/mockSvg.js

```diff
@@ -1 +1 @@
-export default 'svg mock';
+export default 'svg-mock';
```

Comment:
`react-testing-library` throws an "Invalid syntax" error when rendering this, so this fixes it. It doesn't like the space, and the Jest docs also have a hyphenated string.